packages/ember-data/lib/system/model/model.js

```diff
@@ -375,7 +374,7 @@ var Model = Ember.Object.extend(Ember.Evented, {
   */
   toJSON: function(options) {
     // container is for lazy transform lookups
-    var serializer = JSONSerializer.create({ container: this.container });
+    var serializer = this.store.serializerFor('-default');
```

Comment:
no, if you want the record specific serializer, you `record.serialize()`. `toJSON` is using the json serializer for debugging and similar purposes